SharpGrad/Value.cs

```diff
@@ -22,7 +22,8 @@ public Value(double value)
 
             result._backward = () =>
             {
-                // derivative is 1.0. chain rule -> 1.0 * result._grad
+                // y = x + b
+                // dy/dx = 1
```

Comment:
<!-- Comment from Crocodile -->
Comment from [Crocodile 🐊](https://app.crocodile.dev/reviews/rwPWqdvJHMJe?change=SharpGrad%2FValue.cs&iter=1#thread-thQFAgQoPY1A) on unchanged code outside of the pull request:

https://github.com/jlao/SharpGrad/blob/a6b2fd158657e26e99b29ea03d67040387729c43/SharpGrad/Value.cs#L36

do you want to have a '-' here

Reply:
<!-- Comment from Crocodile -->
+1

Posted from [Crocodile 🐊](https://app.crocodile.dev/reviews/rwPWqdvJHMJe?change=SharpGrad%2FValue.cs&iter=1#thread-thQFAgQoPY1A)

Reply:
-1